test/model/dataTypes/FilePath.test.ts

```diff
@@ -0,0 +1,158 @@
+import FilePath from "../../../src/model/dataTypes/FilePath";
+
+describe("FilePath", () => {
+  describe("getInstance()", () => {
+    it("should initialize the path value", () => {
+      expect(FilePath.getInstance("localFile.txt").value).toBe("localFile.txt");
```

Comment:
Codacy found an issue: [Unsafe call of an any typed value.](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5966907)